compiler/module.py

```diff
@@ -0,0 +1,339 @@
+                    (x for x in languages if x["name"] == item["language"]), None
+                )
+                if lang is None:
+                    languages.append(
+                        {
+                            "name": item["language"],
+                            "compilers": [item["name"]],
+                            "templates": item["templates"],
+                        }
+                    )
+                else:
+                    if item["name"] in lang["compilers"]:
+                        await self.log(
+                            level="info",
+                            message=f"Compiler - Load collision: {item['name']}",
+                        )
+                    else:
+                        lang["compilers"].append(item["name"])
+                        lang["templates"].extend(item["templates"])
+                        lang["templates"] = list(set(lang["templates"]))
+            self.languages = sorted(languages, key=lambda d: d["name"])
+
+    def _create_language_embeds(self, ctx: commands.Context) -> List[discord.Embed]:
+        embeds = []
+        for idx, lang in enumerate(self.languages):
+            if idx % 24 == 0:
+                if idx != 0:
+                    embeds.append(embed)  # noqa F821
+                embed = utils.discord.create_embed(
+                    author=ctx.author,
+                    title=_(ctx, "Available languages"),
+                )
+            value = (
+                f"{len(lang['compilers'])} compilers"
+                if len(lang["compilers"]) > 1
+                else f"{len(lang['compilers'])} compiler"
+            )
+            embed.add_field(name=lang["name"], value=value)
+        embeds.append(embed)
+        return embeds
+
+    def _create_language_info_embeds(
+        self, ctx: commands.Context, language: str
+    ) -> List[discord.Embed]:
+        embeds = []
+        comps = [
+            x
+            for x in self.compilers
+            if language.lower()
+            in [
+                x["language"].lower(),
+                x["language"].split(" ")[0].lower(),
+                x["language"].replace(" ", "").lower(),
+            ]
+        ]
+        lang = next(
+            (
+                x
+                for x in self.languages
+                if language.lower()
+                in [
+                    x["name"].lower(),
+                    x["name"].split(" ")[0].lower(),
+                    x["name"].replace(" ", "").lower(),
+                ]
+            ),
+            None,
+        )
+        for idx, comp in enumerate(comps):
+            if idx % 24 == 0:
+                if idx != 0:
+                    embeds.append(embed)  # noqa F821
+                embed = utils.discord.create_embed(
+                    author=ctx.author,
+                    title=_(ctx, "Language compilers"),
+                    description=f"Templates available: `{lang['templates']}`",
```

Comment:
I would replace this line with:
description=f"Templates available: `{'`, `'.join(lang['templates'])}`",

Reply:
This has been resolved